WindowManager/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,26 @@
+<resources>
+    <string name="app_name">WindowSamples</string>
```

Comment:
Ditto, resources tend to be name with undercases, rather than camel case

Reply:
👍